Snippets/Gateway/Gateway_3.2/Gateway_3.2.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net48</TargetFramework>
+    <LangVersion>7.3</LangVersion>
+    <Platforms>x64</Platforms>
```

Comment:
@DavidBoike I see that the other projects also have this, but out of interest, why is it required?